src/features/purchases/screen/orderItem.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { Image, StyleSheet, Text, View } from 'react-native';
+
+import { LineItem } from 'network/models/product-models';
+
+const OrderItem: React.FunctionComponent<{
+  item: LineItem;
+  index: number;
+  size: number;
+}> = ({ item, index, size }) => {
+  return (
+    <View
+      style={[
+        styles.container,
+        index === 0 && styles.containerFirst,
+        index === size - 1 && styles.containerLast,
```

Comment:
When you get the chance to learn about tailwind (https://tailwindcss.com/docs) you will be amazed on how easy this can be solved using just a few classes
https://cruip.com/tailwind-css-nth-child-selector-cheat-sheet/

Reply:
I'm in favor of having these kind of validations outside the style prop, like:
`const isLast = index === size - 1`, then use it in the style prop, with names like `isFirst` or `isLast` is more descriptive for others to know what is happening.